### ISSUE TITLE
Dynamic, Configurable Accessibility Feature Codes

### ISSUE THREAD

**Issue by bredd** [SmarterApp/AdministrationAndRegistrationTool#1]
**Codes and descriptions for accessibility features should be dynamically configurable.**

Accessibility feature codes are presently hard-coded into the applications: ART, Test Delivery, and Reporting. This makes it complicated and expensive add support for new features whenever a new accessibility policy is adopted. Recent history with Smarter Balanced indicates that the Designated Supports and Accommodations, especially non-embedded features, are likely to change in coming years.

Each feature should have a friendly name, a description, and a set of codes for each possible setting. Each feature should be flagged as a universal tool, designated support, or an accommodation. It may not be necessary to define codes for all universal tools but certain tools can be designated-disabled for a student where the tool might be a distraction. So, codes are required for these types.

The set of accessibility features that are available varies between tests -- both by subject matter and by grade level.

Certain features change type from one test to another. In one test it might be a designated support while in another test it might be a universal tool. In those cases, each type of the tool should have its own set of codes.
